doc/src/fix_propel_self.rst

```diff
@@ -8,94 +8,165 @@ Syntax
 
 .. parsed-literal::
 
-   fix ID group-ID propel/self mode magnitude keyword values ...
+   fix ID group-ID propel/self magnitude keyword values
 
 * ID, group-ID are documented in :doc:`fix <fix>` command
 * propel/self = style name of this fix command
-* mode = velocity or quat
-* magnitude = magnitude of the active force
-* one or more keyword/value pairs may be appended to args
-* keyword = *types*
+* magnitude = magnitude of self-propulsion force
+* one (and only one) keyword/value pair must be appended to args
+* keyword = *dipole* or *velocity* or *quat*
+
+  .. parsed-literal::
+
+       *dipole* value = none = apply force along dipole direction
+       *velocity* value = none = apply force along velocity direction
+       *quat* values = direction vector *sx* and *sy* and *sz*
+        *sx* = x component of force direction in ellipsoid frame
+	*sy* = y component of force direction in ellipsoid frame
+	*sz* = z component of force direction in ellipsoid frame
 
-  *types* values = one or more atom types
 
 Examples
 """"""""
 
 .. code-block:: LAMMPS
 
-   fix active_group all propel/self velocity 1.0
-   fix constant_velocity all viscous 1.0
-
-   fix active_group all propel/self quat 1.0
-
-   fix active all propel/self quat 1.0 types 1 2 4
+   fix propel/self all 40.0 dipole
```

Comment:
the syntax example is incorrect here. for all fixes there must be: fix <fix-ID> <fix-group> <fix-name>

Reply:
Thanks for catching this!